trin/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "trin"
+version = "0.1.0"
+authors = ["Jacob Kaufmann <jacobkaufmann18@gmail.com>"]
```

Comment:
Maybe add both of us for the joint one? At some point, this will probably turn into a joint member list, like `trin@ethereum.org`. But for now, we can just list individual maintainers of the individual crates.